### PR TITLE
proposed fix to #39244

### LINKED ISSUE #39244: Python Project - Time Series Visualizer
**Describe your problem and how to reproduce it:**
As mentioned in [this forum post](https://forum.freecodecamp.org/t/page-view-time-series-visualizer-test-error/409465), the tests appear to be missing a flag to allow the dataframe to be converted to an integer.

Whilst the README.md does allude to starting by converting the data to purely `index -> value`, I believe it would be more useful to include the flag:
>Use Pandas to import the data from "fcc-forum-pageviews.csv". Set the index to the "date" column.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-projects/page-view-time-series-visualizer
Project to test: https://repl.it/@Sky020/fcc-time-series-visualizer#test_module.py

**Fix**:
Add the `numeric-only=True` argument to the `df.count()` method.

### PR DESCRIPTION
**Pull request by ricknjacky**
added the numeric-only=True argument to the method as advised

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/39244

### CODE REVIEW

**Comment by RandellDawson** [comment]
@ricknjacky Does this PR related to a reported outstanding issue?  If so, can you please provide the link to the issue?  Thank you.

**Comment by ricknjacky** [comment]
yes @RandellDawson this does correspond to an outstanding issue #39244 link:- [#39244](https://github.com/freeCodeCamp/freeCodeCamp/issues/39244#issue-656076354 )

File: test_module.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 class DataCleaningTestCase(unittest.TestCase):
     def test_data_cleaning(self):
-        actual = int(time_series_visualizer.df.count())
+        actual = int(time_series_visualizer.df.count(numeric-only=True))
```
Comment (ShaunSHamilton) [inline test_module.py]:
```suggestion
        actual = int(time_series_visualizer.df.count(numeric_only=True))
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for this PR. Do note: without this change, the code reads as:
> numeric `<minus>` only `<equals>` True

**Comment by ricknjacky** [comment]
please review @Sky020

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 . Thank you, for making that change.

@RandellDawson Does the workflow for these projects require the _fCC repl.it_ project to be updated? Or, is the boilerplate the only code cloned?

**Comment by scissorsneedfoodtoo** [comment]
Thanks for submitting this PR @ricknjacky, and for your quick review @Sky020. Also, thanks for suggesting the change we'll use for the test @Sky020.

This seems to solve the issue mentioned in https://github.com/freeCodeCamp/freeCodeCamp/issues/39244#issuecomment-702428750. Everything LGTM :+1: :+1: 

I'll go ahead and merge this and make sure the change is reflected on Repl.it, just in case someone forks the boilerplate from there.

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**